client/src/redux/actions/loginActions.js

```diff
@@ -20,14 +20,61 @@ const loginActions = {
                     data: result
                 });
             }, function (result) {
-                console.log("failer");
+                console.log("failure");
                 return dispatch({
                     type: Actions.LOGIN_ERROR,
                     data: result
                 });
             });
         };
-    }
+    },
+
+    forgotPassword: function (email) {
+		return function (dispatch) {
```

Comment:
i think it's safe to assume we'll be supporting modern browsers, lets use ES6 shorthand functions?